kolibri/plugins/learn/assets/src/views/TopicsContentPage.vue

```diff
@@ -134,7 +135,16 @@
         :missingLessonResources="missingLessonResources"
       />
     </SidePanelModal>
-
+    <KModal
+      v-if="showConnectionErrorModal"
+      :title="deviceFormTranslator.$tr('errorCouldNotConnect')"
+      :submitText="coreString('goBackAction')"
+      @submit="goToAllLibraries"
```

Comment:
Is there a case where one piece of content doesn't work but others might from this remote device -- or is this an indication that the remote device cannot serve content to you right now in general?

Reply:
If Kolibri is operating correctly, it should be the latter. At the moment, this is not the case, but I'll be hopefully fixing that in other PRs!